packages/hash/frontend/src/components/layout/PageSidebar/NavLink.tsx

```diff
@@ -1,63 +1,113 @@
-import { useState, FC, ReactNode } from "react";
-import { faChevronRight } from "@fortawesome/free-solid-svg-icons";
-import { Box, Typography, Collapse } from "@mui/material";
+import { useState, FC } from "react";
+import { faAdd, faChevronRight } from "@fortawesome/free-solid-svg-icons";
+import { Box, Typography, Collapse, Tooltip } from "@mui/material";
 import { FontAwesomeIcon } from "../../icons";
-import { IconButton } from "../../IconButton";
-// import { Link } from "./Link";
+import { IconButton, IconButtonProps } from "../../IconButton";
+import { Link } from "../../Link";
 
 type NavLinkProps = {
   title: string;
-  endAdornment: ReactNode;
+  endAdornmentProps: {
+    tooltipTitle: string;
+    "data-testid"?: string;
+    href?: string;
+  } & IconButtonProps;
 };
 
 export const NavLink: FC<NavLinkProps> = ({
   title,
   children,
-  endAdornment,
+  endAdornmentProps,
 }) => {
   const [expanded, setExpanded] = useState(true);
+  const [hovered, setHovered] = useState(false);
+  const {
+    tooltipTitle: endAdornmentTooltipTitle,
+    sx: endAdormentSx = [],
+    href: endAdornmentHref,
+    ...otherEndAdornmentProps
+  } = endAdornmentProps;
+
+  const EndAdornment = (
```

Comment:
Nit: this isn't a react component, so should be camelCase instead of PascalCase.
```suggestion
  const endAdornment = (
```

Reply:
Updated in [ea72a64](https://github.com/hashintel/hash/pull/409/commits/ea72a641d081231e2e1508c51a5bd0ed769a2ec6)

Reply:
This has been addressed as part of https://github.com/hashintel/hash/pull/409/commits/ea72a641d081231e2e1508c51a5bd0ed769a2ec6